src/VIPSimulator.java

```diff
@@ -51,6 +52,8 @@ public static void main(String[] args) {
 		Msg.info("PARAMS:   sostime is " + sosTime + ", number of Gate tasks is " + numberOfGateJobs
 				+ ", number of merge tasks is " + numberOfMergeJobs + ", cpu merge time is " + cpuMergeTime);
 
+		workflowVersion = args.length > 10 ? args[10] : "dynamic";
```

Comment:
maybe use "static" as a default value here, as we cannot fully handle the dynamic case right now. Moreover, with a "static" default we don't have to modify the launchers.